utils/common.sh

```diff
@@ -177,15 +178,16 @@ local METADATA=$(cat << EOF
 "total_nodes":"${TOTAL_NODES}",
 "sdn_type":"${SDN_TYPE}",
 "benchmark":"${BENCHMARK}",
-"start_date":"${START_DATE}",
+"timestamp":"${START_DATE}",
```

Comment:
why are we changing this?

Reply:
We generally use timestamp as date field in our indexes and grafana is only able to handle a time field (Except for uperf which is uperf_ts, that's something we'll have to change to stantarize)

Reply:
ohh right yeah, this was added recently